src/Combobox/ComboboxCategory.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+interface ComboboxCategoryProps {
+  children: React.ReactNode;
+  label?: string;
+}
```

Comment:
React has a helper type PropsWithChildren you might be able to use for many of these
```ts
type ComboboxCategoryProps = React.PropsWithChildren<{
 label?: string;
}>;
```
the only caveat is `children` is optional, which you may not want in certain cases?